tests/test_ble_utils_service_targets.py

```
@@ -0,0 +1,814 @@
+    """Thread probe should normalize invalid fields and swallow probe exceptions."""
+
+    def _raise_alive() -> bool:
+        raise _QueueFailure
+
+    fake_thread = SimpleNamespace(ident="bad-ident", is_alive=_raise_alive)
+    assert ble_utils._thread_start_probe(fake_thread) == (None, False)
+
+
+def test_compatibility_event_service_enqueue_paths() -> None:
+    """Callback enqueue helper should cover non-blocking and fallback queue paths."""
+    callback = lambda: None
```

Ruff (0.15.6)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `callback` as a `def`

(E731)